Documentation/Protocols/NoSQL.md

```diff
@@ -672,7 +672,7 @@ which means that only the `test` database can be accessed and modified.
 
 Since `nosqlprotocol` is a regular protocol module used in a listener,
 the TLS/SSL support of listeners is available. Please see
-[TSLSSL encryption](Getting-Started/Configuration-Guide.md#tsl-encryption)
+[TLSSSL encryption](Getting-Started/Configuration-Guide.md#tls-encryption)
```

Comment:
This technically should be `TLS/SSL` like is written in the line above it. Not necessarily required but if you want to, I'd say you can correct this as well.